tests/conftest.py

```diff
@@ -66,5 +66,6 @@ def login_link(username):
     mail = restmail.get_mail(username)
     mail_content = mail[0]['text'].replace('\n', ' ').replace('amp;', '').split(" ")
     for link in mail_content:
-        if link.startswith("https://auth-dev.mozilla.auth0.com/passwordless/verify_redirect"):
+        if link.startswith("https://auth-dev.mozilla.auth0.com/passwordless/verify_redirect")\
+                or link.startswith("https://auth.mozilla.auth0.com/passwordless/verify_redirect"):
```

Comment:
Rather than list the unique domains, let's use a regular expression or contains:

```python
# regular expression
import re
if re.search('/passwordless/verify_redirect', link)
    return link

# contains
if 'passwordless/verify_redirect' in link:
    return link
```

Another option is to return the first match in `mail_content`:

```python
return next([x for x in mail_content if 'passwordless/verify_redirect' in x])
```